integration_tests/integration/choose-practitioner.spec.ts

```diff
@@ -172,14 +284,15 @@ context('Choose Practitioner', () => {
   })
 
   it('should only be able to select one offender manager at a time', () => {
-    cy.task('stubGetNewToProbationCaseForChoosePractitioner')
     cy.signIn()
     cy.visit('/team/TM1/J678910/convictions/1/choose-practitioner')
     const choosePractitionerPage = Page.verifyOnPage(ChoosePractitionerPage)
-    choosePractitionerPage.radio('OM3').check()
-    choosePractitionerPage.checkedRadioButton().should('have.value', 'OM3')
-    choosePractitionerPage.radio('OM2').check()
-    choosePractitionerPage.checkedRadioButton().should('have.value', 'OM2')
+    choosePractitionerPage.tabtable('all-teams').within(() => {
+      choosePractitionerPage.radio('N03F01::OM1').check()
+      choosePractitionerPage.checkedRadioButton().should('have.value', 'N03F01::OM1')
+      choosePractitionerPage.radio('N03F01::OM1').check()
```

Comment:
the previous test selected a different practitioner whereas this clicks the same one